### DISCUSSION TITLE
Grabaciones Teams

### DISCUSSION THREAD

**Discussion by Luis234** [Launch-X-Latam/MisionFrontEnd#164] [Q&A]
Donde se quedan grabadas las sesiones que hemos tenido? gracias :)

**Comment by HansonAlejandro** [comment]
Hola Luis, una vez entrando al equipo de microsoft teams, te vas al apartado equipos, seleccionas launch x, y en la parte inferior dice canales, ahí aparecen las diversas clases separadas por módulos, Saludos

**Comment by David-Carr-C** [comment]
En mi computadora puedo ver la charla a través del mismo link de la transmisión, pero otras formas son en Grabaciones que se localiza en General>Más>Grabaciones

O también en los anuncios al final de la semana donde ponen los enlaces de transmisiones grabadas.

**Comment by doguedogue** [comment]
https://web.microsoftstream.com/user/96eeb66f-b984-4083-8ea6-74c34a07a34a
